### ISSUE TITLE
[Feature request] List depots for a given appID or manifests for a given appID & depotID

### ISSUE THREAD

**Issue by NotWearingPants** [SteamRE/DepotDownloader#260]
Instead of looking for the IDs using another tool/website, it would be cool for this CLI to have some `-list` parameter to be able to show a table of all available IDs to choose from, along with metadata such as size/date/OS/etc.

Basically the same thing as [this](https://steamdb.info/app/570/depots/) and [this](https://steamdb.info/depot/373303/manifests/).

**Comment by nutterthanos** [comment]
yeah since steamdb is like the only site that has if not all the manifests for nearly every app/game but that can't really get used unless thhe info gets cached in a .json file or something

**Comment by azuisleet** [comment]
The only data available to be listed is the current appinfo.

**Comment by nutterthanos** [comment]
i mean https://github.com/ValvePython/steamctl/ can already pretty much do what you want ig this

**Comment by nutterthanos** [comment]
like this 
![image](https://user-images.githubusercontent.com/50470661/135790303-84847720-83db-4665-96e3-32e57bb3e775.png)
